app/init.js

```diff
@@ -1,27 +1,21 @@
-const utils = require('@/utils');
-const buoys = require('@/routes/erddap/utils');
-const mcache = require('memory-cache');
+const utils = require("@/utils");
+const buoys = require("@/routes/erddap/common");
+const mcache = require("memory-cache");
 
 const updateCache = async () => {
   const end = new Date().toISOString(); // now
-  const key = `__express__/erddap/buoy/summary`;
-  const ids = buoys.ids;
-  const variable = buoys.variables.join(',');
-  console.log(`initialize cache for ${key}`);
-
-  const payload = {
-    ids,
-    variable,
-    datasetId: 'combined_e784_bee5_492e',
-    start: '2000-07-01T12:00:00Z',
-    end
+  const keyBase = `__express__/erddap`;
+  const initMap = {
+    buoy: "combined_e784_bee5_492e",
```

Comment:
Maybe these initial buoy and model ids as well as the combined `DATASET_ID` in `buoy.js`  could be all abstracted together